package.json

```diff
@@ -24,7 +24,7 @@
   "devDependencies": {
     "grunt": "^1.0.1",
     "grunt-contrib-cssmin": "^2.2.0",
-    "grunt-contrib-uglify": "^3.0.1",
+    "grunt-contrib-uglify": "gruntjs/grunt-contrib-uglify#harmony",
```

Comment:
The branch is held up to date to the master tree, but I think there is no official release available yet.